app/routes/forum/components/ForumList.tsx

```diff
@@ -30,11 +33,21 @@ const ForumList = () => {
         )
       }
     >
-      <Flex column>
-        {forums?.map((f: PublicForum) => (
-          <ForumListEntry forum={f} key={f.id} />
-        ))}
-      </Flex>
+      <Helmet title="Forum" />
+      {!forums.length && (
+        <EmptyState
+          iconNode={<MessageSquareDashed />}
+          header="Her var det tomt ..."
+          body="Opprett et nytt forum da vel!"
+        />
+      )}
+      {forums.length > 0 && (
+        <Flex column>
+          {forums?.map((f: PublicForum) => (
```

Comment:
```suggestion
          {forums?.map((forum: PublicForum) => (
```
Not your doing, but spelling it out would be kinda nice(:

Reply:
Fair enough! Fixed 🦖